smoke-tests-login.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import os
+import urllib
+import requests
+
+if __name__ == "__main__":
+    session = requests.Session()
+
+    logs = session.get('https://logs.{}'.format(os.environ['CF_SYSTEM_DOMAIN']))
+    print(logs.url, logs.status_code)
+    assert logs.status_code == 200
```

Comment:
Shouldn't this be a 302 to UAA / can we assert we are being redirected to login.CF_SYSTEM_DOMAIN?